qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -267,6 +267,8 @@ def parse_offset_get(s):
 
         self.add_function('disable_front_panel', call_cmd='OVRM 0')
         self.add_function('enable_front_panel', call_cmd='OVRM 1')
+        
+        self.add_function('id', call_cmd='*IDN?')
```

Comment:
I guess  you want this just to get the full idn string, instead of the parsed one from get_idn ?

I kind of think it would be nicer if the name was idn, what  do you think ?